### ISSUE TITLE
Sounding

### ISSUE THREAD

**Issue by mbeaufils** [opengeospatial/Geotech#37]
Do we need this concept?
Proposed definition.

**Comment by dponti** [comment]
Yes, I think so. A sounding is similar to a borehole in that it is a 1D spatial sample and it would inherit its geometry properties from the same "parent" class. It is also constructed, like a borehole, but would not carry the same construction properties (or as many). It is also unique (at least as I consider this object) in that is is created as part of an Observation (a measurement, actually), so there is some value in differentiating a sounding from a borehole. 

Proposed definition: A sounding is a type of spatial sample that is created by an observation (activity) that involves the insertion of a probe into the ground. 

BTW, one might think that a sounding only exists as part of a measurement, but I was actually involved in project where piezometers were installed in the opening created by the CPT probe. One can also extract samples (for additional observations) as part of the sounding process...

**Comment by mbeaufils** [comment]
I have the impression that:
- Borehole is a "nature driven" wording,
- Sounding is a "usage or purpose driven" wording. Just like well.
